packages/promotion/src/models/application-method.ts

```diff
@@ -42,6 +44,12 @@ export default class ApplicationMethod {
   @Property({ columnType: "numeric", nullable: true, serializer: Number })
   max_quantity?: number | null
 
+  @Property({ columnType: "numeric", nullable: true, serializer: Number })
+  apply_to_quantity?: number | null
+
+  @Property({ columnType: "numeric", nullable: true, serializer: Number })
+  buy_rules_min_quantity?: number | null
```

Comment:
suggestion: Convert to
```suggestion
  @Property({ columnType: "numeric", nullable: true, serializer: Number })
  apply_to_quantity: number | null = null

  @Property({ columnType: "numeric", nullable: true, serializer: Number })
  buy_rules_min_quantity: number | null = null
```

Reply:
And rest of model

Reply:
changed in all models in the module.

Reply:
@olivermrbl also added some validations for the API in the new commit, take a look at those as well when you re-review.